### DISCUSSION TITLE
Disable countdown?

### DISCUSSION THREAD

**Discussion by JoeSz** [evilsocket/opensnitch#581] [General]
Is there an Option to disable countdown?
I would like to disable countdown, so the GUI window does not disappear until I press allow or deny.

**Comment by gustavo-iniguez-goya** [comment]
Hi @JoeSz !

If you mean disable the timeout but show pop-ups , it's not possible right now. There's an internal hard-coded maximum timeout to answer a pop-up (120s). On the other hand, on v1.4.x the countdown is stopped if you click on the `[+]` button. On v1.5.x it's disabled if you click in any button except Deny or Allow.

If you want to disable the countdown, you can do it under `Preferences -> Pop-ups -> [x] Disable pop-ups, only display an alert`
But it won't display the pop-up.

**Comment by JoeSz** [comment]
Thank you for the quick answer!

"If you mean disable the timeout but show pop-ups , it's not possible right now. "
May I ask why?
(Or should I ask this in the feature request? :) )

I don't think (as a programmer) it would be really difficult to implement, for example with a checkbox.
And also it would be very useful. For example, I am, for whatever reason, not sitting in front of the computer when a request comes in.
